lib/r10k/api.rb

```diff
@@ -212,5 +259,54 @@ def deploy_module_into_env(module_name, env_map, path, opts={})
 
     private
 
+
+    def parse_deployed_git_env(path, moduledir)
+      env_repo = R10K::Git.provider::WorkingRepository.new(path, '')
+
+      env_data = {
+        :source => {
+          :type => :git,
+          :remote => env_repo.origin,
+          :branch => nil, # TODO: store and extract from .r10k-deploy.json?
+        },
+        :version => env_repo.head,
+        :resolved_at => nil, # TODO: Extract from .r10k-deploy.json? or current time?
+      }
+
+      # Add :modules declared in Puppetfile
+      env_data.merge!(self.parse_puppetfile(File.join(path, "Puppetfile")))
+
+      # Find the deployed version of each module.
+      env_data[:modules].map! do |mod|
+        mod_path = File.join(path, moduledir, mod[:name])
+        mod.merge(parse_deployed_module(mod_path, mod[:type]))
+      end
+
+      return env_data
+    end
+
+    def parse_deployed_svn_env(path, moduledir)
```

Comment:
I kind of want to raise a NotImplementedError here, just to be able to trace what happened easier if this ever comes up.